src/coreanimation.cs

```diff
@@ -584,6 +586,35 @@ interface CALayer : CAMediaTiming, NSSecureCoding {
 		[TV (11,0)][Mac (10,13)][iOS (11,0)]
 		[Export ("maskedCorners", ArgumentSemantic.Assign)]
 		CACornerMask MaskedCorners { get; set; }
+
+		[BindAs (typeof (CACornerCurve))]
+		[NoWatch] // headers not updated
+		[TV (13,0)][Mac (10, 15, onlyOn64: true)][iOS (13, 0)]
+		[Export ("cornerCurve")]
+		NSString CornerCurve { get; set; }
+
+		[NoWatch] // headers not updated
+		[TV (13,0)][Mac (10, 15, onlyOn64: true)][iOS (13, 0)]
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Static]
+		[Export ("cornerCurveExpansionFactor:")]
+		nfloat GetCornerCurveExpansionFactor (NSString curve);
```

Comment:
Why not `nfloat GetCornerCurveExpansionFactor ([BindAs (typeof (CACornerCurve))] NSString curve);` instead of the manual wrap? Since you are already doing it for the property.

Reply:
I forgot about it - and updated the others afterward (and missed that one) :D